ta4j-core/src/main/java/org/ta4j/core/Backtesting.java

```diff
@@ -0,0 +1,61 @@
+package org.ta4j.core;
+
+import org.ta4j.core.num.Num;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This class enables backtesting of multiple strategies and comparing them to see wich is the best
+ */
+public class Backtesting {
+
+    private TimeSeries seriesToTradeOn;
+    private TimeSeriesManager seriesManager;
+    private List<Strategy> strategies = new ArrayList<>();
+    private List<BacktestingResult> backtestingResults = new ArrayList<>();
+
+    public Backtesting(TimeSeries series) {
+        this(series, series);
+    }
+
+    public Backtesting(TimeSeries seriesToBacktest, TimeSeries seriesToTradeOn) {
+        this.seriesToTradeOn = seriesToTradeOn;
+        this.seriesManager = new TimeSeriesManager(seriesToBacktest);
+    }
+
+    public void addStrategy(Strategy strategy) {
+        this.strategies.add(strategy);
+    }
+
+    public List<BacktestingResult> getBacktestingResults() {
+        return backtestingResults;
+    }
+
+    /**
+     * Calculate your strategies and giving you a backtesting result
+     *
+     * @param amount    - The amount used to open/close the trades
+     * @param priceType - The type of price used in calculation (open, high, low, close)
+     */
+    public List<BacktestingResult> calculate(Num amount, PriceType priceType) {
+        for (Strategy strategy : strategies) {
+            TradingRecord tradingRecord = seriesManager.run(strategy, Order.OrderType.BUY, amount, priceType);
```

Comment:
I think it should be possible to specify the `OrderType`

Reply:
Fixed. What is this OrderType beeing used to?